### DISCUSSION TITLE
macOs - Behind corporate firewall, SSL cert installation

### DISCUSSION THREAD

**Discussion by anantyadunath** [containers/podman#24539] [Q&A]
Hi community,

I am new to the community and to Podman on macOS. I am primarily a Windows user however recently trying to setup Podman for Mac, so please bear with me if the questions are simple / lame.

I am trying to install Podman v5.2.5 on macOS which is behind Zscaler corporate network. I have installed and trusted all organizational SSL certs provided on the Mac machine and I do not face any SSL issues when it comes to Mac. However when I try to execute any commands on the "Podman machine" (for example curl) it throws an SSL error since the certificate is a MIM from Zscaler self signed by my org.

I faced similar issues when setting up for windows, in which case, I used to install my SSL certs inside Podman machine and everything works. I assume that on windows, it's the plain old Fedora installation instead of Fedora CoreOS and I am struggling to install the certs in this setup. I already tried def, however it is missing. I tried rpm-ostree, however do not understand how to use it correctly since seems that the actual podman machine is a virtual machine inside CoreOS.

Any guidance  in this context would help. Thanks !

**Comment by Luap99** [comment]
https://github.com/containers/podman/blob/main/docs/tutorials/podman-install-certificate-authority.md
